### DISCUSSION TITLE
Is there a containerized version of LISF 7.5?

### DISCUSSION THREAD

**Discussion by ShangyongShi** [NASA-LIS/LISF#1619] [Q&A] [answered]
Hi,

I am new to LIS. I haven't got my access to Discover yet, so I need to work on our group server. I am wondering if there is a containerized version of LISF 7.5 for a quicker start?

Thanks.

**Comment by jvgeiger** [comment, marked as answer]
Yes.  Click on Releases.  You will find an AppImage container for the latest public release, 7.5.1, and for the master branch.

**Reply by bayoleems** [reply]
Hi @jvgeiger, I'm experiencing issues with running the public test cases for LISF (LIS, LDT, and LVT) using the AppImage. Despite successfully completing Step 1, I'm encountering errors in the `log/lislog_ol.0000` file when running Step 2 test cases. 

<img width="1455" alt="Screenshot 2025-04-20 at 4 38 43 PM" src="https://github.com/user-attachments/assets/946ee161-e458-469e-adcd-e853242df14b" />

Any guidance or suggestions would be greatly appreciated!

**Reply by dmocko** [reply]
Hi @bayoleems 

Can you please let us know if you are using the latest public release or the master branch AppImage?

We may need to update our public testcase to be consistent with the data available on the GES DISC,
and with the latest codes.

**Reply by bayoleems** [reply]
Hi @dmocko 

Thank you for your response. I'm currently using LISF-x86_64.AppImage for LISF Public 7.5.2 (Dec 11, 2024) and the LIS Public Testcase Walkthrough for LISF Public 7.4 as a guide. I noticed there's a pre-release AppImage available. Would you recommend using that instead? I'm looking forward to the updated public test case documentation. Thanks!

**Comment by jvgeiger** [comment]
Hello,

I do not recommend trying the pre-release AppImage right now because that will require tweaking the lis.config file for the public testcase.  First, did you get any errors when running the wget_gesdisc_nldas2.sh script?  Your error message suggests that you are missing some NLDAS2.FORCING files.


Thanks,
Jim

**Reply by bayoleems** [reply]
Hi @jvgeiger 

Yes I did get errors from downloading the files from NLDAS directory but these files are not found on the interface as well. 

Here is a log from the wget_gesdisc_nldas2.sh script

![image](https://github.com/user-attachments/assets/981e708b-fe08-413e-92f5-22c7f4f8129c)

**Reply by karsenau** [reply]
Hi @bayoleems and @jvgeiger:

The "wget_gesdisc_nldas2.sh" script only supports former grib format downloads from NASA GES-DISC. However, they recently removed the Grib formatted from their services and have now moved to NetCDF format.

To work with our LIS public set of tutorial testcases, most likely GRIB-1 files will need to be provided, since the script you have will not work any longer, and the LIS version you may be working with may not support the netcdf format.  

@dmocko:  Does the latest LIS 7.5 public version support the latest NLDAS2 netcdf format?

Will follow-up in the next couple of days with an update to this situation. Thank you.

**Reply by bayoleems** [reply]
Hi @karsenau 

Here,

Step 2: LSM "Open-loop" (OL) Experiment (LIS), Download the Meteorological Forcing Dataset.

https://nasa-lis.github.io/LISF/public_testcase_walkthrough/public_testcase_walkthrough.html

The lis test case zip folder had the script.

**Reply by karsenau** [reply]
@bayoleems:

Thank you for confirming that.  Please see my edited reply above.

Thank you.

**Reply by bayoleems** [reply]
Alright, thank you.

@karsenau, what other test cases do you advise I familiarize myself with that have no recent changes?

**Comment by karsenau** [comment]
@bayoleems:

We are actively working on a new public code update and will update parts of the testcases and wget script for the netCDF data.

Thanks for your patience.
